chase/frontend/components/dashboard/documents/draft.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import DocumentTemplate from "@components/dashboard/documents/documenttemplate";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFileLines } from "@fortawesome/free-solid-svg-icons";
+import Image from "next/image";
+
+export default function Draft({
+  number,
+  introducedBy,
+  sponsors,
+}: { number: string; introducedBy: string; sponsors: string[] }) {
+  return (
+    <DocumentTemplate>
+      <FontAwesomeIcon icon={faFileLines} className="text-gray-400 text-2xl" />
+      <div className="flex-1 flex-col justify-start items-center">
+        <div className="text-sm font-semibold text-gray-600">{number}</div>
```

Comment:
Icons are good but I'd suggest prefixing the document name with its type. E.g. "Draft: yxz"